lib/mistral_ai_summary_example/summary_settings_page.dart

```diff
@@ -11,30 +12,31 @@ class SettingsWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Theme.of(context).colorScheme.surfaceVariant,
-      appBar: AppBar(
-        title: const Text('Settings'),
-        backgroundColor: Theme.of(context).colorScheme.surfaceVariant,
-      ),
-      body: Column(
-        children: [
-          Expanded(
-            child: ListView(
-              children: const [
-                ModelSettingWidget(),
-                SettingsListSpacer(),
-                TemperatureAndTopPSettingWidget(),
-                SettingsListSpacer(),
-                MaxTokensSettingWidget(),
-                SettingsListSpacer(),
-                RandomSeedSettingWidget(),
-                SettingsListSpacer(),
-                SafePromptSettingWidget(),
-              ],
+    return DarkerBackgroundTheme(
+      child: Scaffold(
+        appBar: AppBar(
+          title: const Text('Settings'),
+          backgroundColor: Theme.of(context).colorScheme.surfaceVariant,
```

Comment:
I think with `DarkerBackgroundTheme` you can skip setting color here.

Reply:
Resolved in [a9515c3](https://github.com/nomtek/flutter_ai_examples/pull/30/commits/a9515c3056e0818ada1df147572f4ba6f5c2442b)